EIPS/eip-4430.md

```diff
@@ -0,0 +1,154 @@
+---
+eip: 4430
+title: Described Transactions
+description: Contract methods to compute human-readable descriptions for machine readable transaction data.
+author: Richard Moore (@ricmoo), Nick Johnson (@arachnid)
+discussions-to: https://github.com/ethereum/EIPs/issues/4431
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-11-07
+---
+
+## Simple Summary
+
+A technique for contract authors to enable wallets to provide a
+human-readable description of the effect of of a transaction with a
+given contract.
+
+
```

Comment:
```suggestion
```

Reply:
Is this suggestion that the "simple summary" section should be removed?

Reply:
Yeah.  We have removed it from the EIP template in favor of a `description` field in the frontmatter.

Reply:
Cool cool. I realized what you did after that command and I moved it. :)